docs/geode-native-book/redirects.rb

```diff
@@ -15,6 +15,8 @@
 
 # Links to API Documentation #
 r301 %r{/releases/latest/javadoc/(.*)}, 'https://geode.apache.org/releases/latest/javadoc/$1'
+r302 %r{/cppdocs/(.*)}, 'https://gemfire-nc-100-cppdocs.docs.pivotal.io/$1'
```

Comment:
Is this the right url?  Was expecting something on `geode.org`

Reply:
Good catch. Thanks!